codec-http2/src/main/java/io/netty/handler/codec/http2/AbstractHttp2StreamFrame.java

```diff
@@ -22,17 +22,18 @@
  */
 @UnstableApi
 public abstract class AbstractHttp2StreamFrame implements Http2StreamFrame {
-    private Object stream;
+
+    private int streamId = -1;
 
     @Override
-    public AbstractHttp2StreamFrame setStream(Object stream) {
-        this.stream = stream;
+    public AbstractHttp2StreamFrame setStreamId(int streamId) {
```

Comment:
@buchgr do we need todo some validation ? Like streamId can only be positive etc ?